### DISCUSSION TITLE
Rview Should not be handeled by vendors/sellers - Fluxstore  Manager App ✅

### DISCUSSION THREAD

**Discussion by HarrySmith01** [inspireui/support#5082] [Working on Features]
In whole world, not a single ecommerce market allows sellers to manage reviews & ratings. it is not just because it is a habit or culture coming from the long time but it is being if we allow seller to manage reviews then non of their product will ever have negative reviews. Marketplace will be full of positive reviews.

This is not just dangerous feature for a marketplace & customer but it is illegal too.

**Comment by sam-shl** [comment]
هذه المذكرة مهمة جدا
ميزة التعليقات عديمة الفائدة عندما يتحكم البائعون فيها

**Comment by seb1911** [comment]
Am I the only one to see that FluxStore has many issues with the Reviews ?

**Comment by philipvu30** [comment]
Hi,

You can simply remove this code and the user won't be able to modify the review. https://tppr.me/MDl2V
It's in this file lib/frameworks/vendor_admin/screens/review_approval_screen/widgets/review_item.dart

The reason why we enabled it in the first place that we aim for WooCommerce platform but we switch to WCFM first while developing this app. We'll take note of this and add an option to enable/disable it in the project in the version 2.2.

Our apologies for any inconvenience and thank you for your suggestion.
